README.md

```diff
@@ -108,6 +108,10 @@ for musl libc.
 Linux toolchain has an additional option `--enable-default-pie` to control the
 default PIE enablement for GCC, which is disable by default.
 
+To customize the enabled languages, use option `--with-languages=`. For example, 
+if you want to enable `c,c++,fortran`, use `./configure --with-languages=c,c++,fortran`.
+This option only takes effect for the GNU toolchain.
```

Comment:
Shouldn't this be "use `./configure --with-languages="c,c++,fortran"`? I.e. the double quotes are missing?

Reply:
Also works without the quotes.

Reply:
Ah - ok - I assumed that quotes were mandatory and the comma might be interpreted by the command shell etc.